src/apis/sourceimages.js

```diff
@@ -137,13 +137,19 @@ export default (state) => {
    *   .catch(function(err) {});
    * ```
    *
+   * With directly adding metadata:
+   *
+   * rokka.sourceimages.create('myorg', 'picture.png', file, {'meta_user': {'foo' => 'bar'}})
```

Comment:
can you add triple backticks around the code?

Reply:
added